cmd/wallet-sdk-gomobile/credentialquery/credentialquery.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/piprate/json-gold/ld"
 
 	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-gomobile/api"
-	"github.com/trustbloc/wallet-sdk/internal/gomobilewrappers"
+	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-gomobile/gomobilewrappers"
```

Comment:
No longer internal? Was there some issue you ran into? Those wrappers will appear in the bindings if it's not internal

Reply:
if it is internal in the outside package we have circular dependencies. root package began to depend on cmd.
If make it internal inside cmd it will break the mobile bindings build.